awx/ui/client/features/projects/projectsList.controller.js

```diff
@@ -46,8 +47,49 @@ function projectsListController (
         } else {
             vm.activeId = '';
         }
+        setToolbarSort();
     }, true);
 
+    function setToolbarSort () {
+        const orderByValue = _.get($state.params, 'project_search.order_by');
+        const sortValue = _.find(vm.toolbarSortOptions, (option) => option.value === orderByValue);
+        if (sortValue) {
+            vm.toolbarSortValue = sortValue;
```

Comment:
https://github.com/ansible/awx/pull/3356#issuecomment-469363804

This should set the Sort dropdown to the current value of `$state.params.order_by`.